packages/cli/scripts/util/cdkHelpers.js

```diff
@@ -193,13 +193,17 @@ async function loadEsbuildConfigOverrides(configPath) {
   }
   const configOverrides = require(configFullPath);
 
-  // validate only "plugins" can be overrid
+  const allowedOptions = ["plugins", "keepNames"];
+
+  // validate only allowed options can be overridden
   const nonPluginsKey = Object.keys(configOverrides).find(
```

Comment:
Let's rename `nonPluginsKey` to something more descriptive ie. `unallowedKey`

Reply:
Sorted